engine/executor/dm/worker.go

```diff
@@ -187,15 +199,17 @@ func (w *dmWorker) persistStorage(ctx context.Context) error {
 
 // tryUpdateStatus updates status when task stage changed.
 func (w *dmWorker) tryUpdateStatus(ctx context.Context) error {
-	stage := w.unitHolder.Stage()
-	if stage == w.getStage() {
+	stage, _ := w.unitHolder.Stage()
+	currentStage := w.getStage()
```

Comment:
I forget why we have to keep 2 stage. Please left some comments

Reply:
rename variable